emscripten.py

```diff
@@ -1764,6 +1764,8 @@ def build_wasm(temp_files, infile, outfile, settings, DEBUG):
     # Also convert wasm text to binary
     wasm_as_args = [os.path.join(shared.Settings.BINARYEN_ROOT, 'bin', 'wasm-as'),
                     wast, '-o', basename + '.wasm']
+    if settings['DEBUG_LEVEL'] >= 2 or settings['PROFILING_FUNCS']:
```

Comment:
`test_demangle_stacks`, for instance, only uses `--profiling-funcs` instead of passing in `-g`.

When we invoke asm2wasm, we use the same logic of `debug_level >= 2 or profiling_funcs`, https://github.com/kripken/emscripten/blob/incoming/emcc.py#L2241 and https://github.com/kripken/emscripten/blob/incoming/emcc.py#L2260 , so this matches that.